### ISSUE TITLE
对于atepc关于使用自己的数据集APC指标较低的问题？

### ISSUE THREAD

**Issue by zhujinqiu** [yangheng95/PyABSA#124]
我标注了一套自己的数据集，单独跑apc（用你们的APC模型跑的）任务指标正常。APC acc为91，f1为91.但是我跑多任务的时候，用你们的ATEPC的时候，ATE指标倒是正常，可是APC的指标很低，F1为37.37(max:37.37)。。。这是为啥。。。
是不是我想做多任务的时候只能先方面抽取ATE再情感极性分类？

**Comment by yangheng95** [comment]
请问你使用的模型是哪一个，config能提供一下吗

**Comment by zhujinqiu** [comment]
> 请问你使用的模型是哪一个，config能提供一下吗

BERT Base--ATEPC 还有lcf-ATEPC都很差，参数用的atepc_config = ATEPCConfigManager.get_atepc_config_english()
atepc_config.model = ATEPCModelList.BERT_BASE_ATEPC
atepc_config.num_epoch = 10
# atepc_config.model = ATEPCModelList
atepc_config.evaluate_begin = 0
atepc_config.log_step = 100
atepc_config.dropout = 0.1
atepc_config.l2reg = 0.00005
atepc_config.batch_size = 16
atepc_config.max_seq_len = 120
atepc_config.srd_alignment = False
都是默认的，如果不跑多任务，单独跑APC任务，APC精度就正常。。。跑多任务精度一直是37.37很奇怪。

**Comment by yangheng95** [comment]
请问使用的预训练模型是bert-base-uncased吗

**Comment by zhujinqiu** [comment]
> 请问使用的预训练模型是bert-base-uncased吗

atepc_config.pretrained_bert = 'bert-base-uncased'是的

**Comment by yangheng95** [comment]
我在其他数据集上跑没有问题，你看是不是数据标错了。之前也有很多issue都是因为数据标的有问题

**Comment by yangheng95** [comment]
你可以提供一个数据集切片好分析

**Comment by zhujinqiu** [comment]
> 请问你使用的模型是哪一个，config能提供一下吗

[你可以看看这个数据集？](https://github.com/zhujinqiu/test_data)里面2个文件，你可以跑跑apc和atepc。。单跑APC精度是正常的，但是ATEPC里的APC精度很奇怪。。。

**Comment by yangheng95** [comment]
你好，
我测试之后没有发现问题，测试版本1.6.16
![image](https://user-images.githubusercontent.com/51735130/147663811-3ca3d67d-9a4d-43e5-8d48-b2aa521a22fd.png)
顺便请问您，是否愿意在版权允许的情况下合并数据集到ABSADatasets以供研究？

**Comment by zhujinqiu** [comment]
> 你好， 我测试之后没有发现问题，测试版本1.6.16 ![image](https://user-images.githubusercontent.com/51735130/147663811-3ca3d67d-9a4d-43e5-8d48-b2aa521a22fd.png) 顺便请问您，是否愿意在版权允许的情况下合并数据集到ABSADatasets以供研究？

能提供下你的设置参数吗。。。我自己跑的有点问题。估计是之前跑你们内部debug的时候，改到什么部分了，待会重新更新下版本。
关于合并问题。。后期我论文发了再给你们合并吧，我最近在搞小论文。毕竟你们这个Repo帮了我很多，还有之前那位大佬疯狂帮我debug标注工具，真的很感谢。所以后面论文出来之后，数据集和你们共享

**Comment by yangheng95** [comment]
```
########################################################################################################################
#                                               ATEPC training script                                                  #
########################################################################################################################
from pyabsa.functional import ATEPCModelList
from pyabsa.functional import Trainer, ATEPCTrainer
from pyabsa.functional import ABSADatasetList
from pyabsa.functional import ATEPCConfigManager

atepc_config = ATEPCConfigManager.get_atepc_config_english()
atepc_config.num_epoch = 10
atepc_config.model = ATEPCModelList.BERT_BASE_ATEPC
atepc_config.evaluate_begin = 2
atepc_config.log_step = 100
atepc_config.dropout = 0.1
atepc_config.l2reg = 0.00005
atepc_config.batch_size = 32
atepc_config.srd_alignment = False

atepc_config.pretrained_bert = 'bert-base-uncased'
atepc_config.model = ATEPCModelList.FAST_LCF_ATEPC
dataset_path = ABSADatasetList.Restaurant14
dataset_path = 'openreview'
# or your local dataset: dataset_path = 'your local dataset path'

aspect_extractor = ATEPCTrainer(config=atepc_config,
                                dataset=dataset_path,
                                from_checkpoint='',  # set checkpoint to train on the checkpoint.
                                checkpoint_save_mode=1,
                                auto_device=True
                                ).load_trained_model()
```

**Comment by yangheng95** [comment]
> > 你好， 我测试之后没有发现问题，测试版本1.6.16 ![image](https://user-images.githubusercontent.com/51735130/147663811-3ca3d67d-9a4d-43e5-8d48-b2aa521a22fd.png) 顺便请问您，是否愿意在版权允许的情况下合并数据集到ABSADatasets以供研究？
> 
> 能提供下你的设置参数吗。。。我自己跑的有点问题。估计是之前跑你们内部debug的时候，改到什么部分了，待会重新更新下版本。 关于合并问题。。后期我论文发了再给你们合并吧，我最近在搞小论文。毕竟你们这个Repo帮了我很多，还有之前那位大佬疯狂帮我debug标注工具，真的很感谢。所以后面论文出来之后，数据集和你们共享

明白！欢迎以后共享数据集模型或者提供其他帮助，代表社区感谢您
